wagtailio/showcase/models.py

```diff
@@ -0,0 +1,176 @@
+from django.core.exceptions import ValidationError
+from django.core.paginator import EmptyPage, PageNotAnInteger, Paginator
+from django.db import models
+
+from modelcluster.fields import ParentalKey
+from wagtail.admin.panels import FieldPanel, InlinePanel, MultiFieldPanel
+from wagtail.fields import StreamField
+from wagtail.models import Orderable, Page
+from wagtail.search import index
+
+from wagtailio.core.blocks import StandaloneCTABlock
+from wagtailio.showcase.snippets import Sector
+from wagtailio.utils.models import CrossPageMixin, SocialMediaMixin
+
+
+class ShowcasePage(SocialMediaMixin, CrossPageMixin, Page):
+    template = "patterns/pages/showcase_page/showcase_page.html"
+    htmx_template = "patterns/pages/showcase_page/showcase_page_fragment.html"
+
+    parent_page_types = ["core.HomePage"]
+
+    introduction = models.TextField(
+        verbose_name="Introduction",
+        blank=True,
+    )
+    cta = StreamField(
+        [("cta", StandaloneCTABlock())],
+        blank=True,
+        max_num=1,
+        help_text="Allows for a maximum of 1 CTA blocks",
+        use_json_field=True,
+    )
+
+    listing_meta_description = models.CharField(
+        max_length=255, blank=True, help_text="The description beneath the listing"
+    )
+
+    content_panels = Page.content_panels + [
+        FieldPanel("introduction"),
+        FieldPanel("cta"),
+        InlinePanel("showcase_items", label="Showcase items"),
+    ]
+
+    promote_panels = (
+        Page.promote_panels + SocialMediaMixin.panels + CrossPageMixin.panels
+    )
+
+    search_fields = Page.search_fields + [
+        index.SearchField("introduction"),
+    ]
+
+    def _filter_used_sectors(self):
```

Comment:
nitpick: this reads like an action to me. How about something like `_get_used_sectors()` and drop the "All" value (or better said, shift "All" to `get_context`)

Reply:
Done, I've also extended the method slightly due to handling the FakeQuerySet in the preview.